docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,108 +1,75 @@
 [[connect-to-elasticsearch]]
 == Add data to {kib}
+++++
+<titleabbrev>Add data</titleabbrev>
+++++
 
-To start working with your data in {kib}, you can:
+The fastest way to add data to {kib} is to use one of the built-in options.
```

Comment:
Thoughts on replacing "use one of our built in options" with "one of our many ingest options"?